File: app/src/main/java/com/jeff/dytbr/MyWebChromeClient.java
package com.jeff.dytbr;

import android.app.Activity;
import android.util.Log;
import android.webkit.WebChromeClient;
import android.webkit.WebView;
import android.widget.ProgressBar;

public class MyWebChromeClient extends WebChromeClient {

    ProgressBar myProgressBar;

    public MyWebChromeClient(Activity myActivity) {
        this.myProgressBar = myActivity.findViewById(R.id.myProgressBar);
    }
    
    @Override
    public void onProgressChanged(WebView view, int newProgress) {
        super.onProgressChanged(view, newProgress);
        myProgressBar.setProgress(newProgress);
        Log.i("PROGRESS" , "progress: " + newProgress);
    }
}

File: app/src/main/java/com/jeff/dytbr/MyWebChromeClient.kt
package com.jeff.dytbr

import android.app.Activity
import android.util.Log
import android.webkit.WebChromeClient
import android.webkit.WebView
import android.widget.ProgressBar

class MyWebChromeClient(myActivity: Activity) : WebChromeClient() {
    var myProgressBar: ProgressBar

    init {
        myProgressBar = myActivity.findViewById(R.id.my_progressBar)
    }

    override fun onProgressChanged(view: WebView, newProgress: Int) {
        super.onProgressChanged(view, newProgress)
        myProgressBar.progress = newProgress
        Log.i("PROGRESS", "progress: $newProgress")
    }
}